sprint0/sprint0/Game1.cs

```diff
@@ -10,6 +10,7 @@ public class Game1 : Game
         private readonly GraphicsDeviceManager _graphics;
         private SpriteBatch _spriteBatch;
         private List<IController> controllerList;
+        public List<int> VisitedRooms;
```

Comment:
The game class is getting kind of long, could we have a screen manager class for all the different screens. I personally don't care but I feel like the grader will